### DISCUSSION TITLE
2 channel encoder vs 1 channel encoder  (sbc)

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#804] [Q&A] [answered]
I notice encoder 2 channel uncompressed byte =512 and compressed =76 
while 1 channel encoder uncompressed bytes = 256 and compressed = 72

which means 512 bytes uncompressed will compressed to 76 bytes for 2 channel data and 256 to 72 for 1 channel data right ? 

and at out put if i send compressed bytes to espnow or for further processing their is note huge difference right ? out put packet will only 4 byte less as compare to 2 channel out put right ? 

I was expecting out put packet will also reduce half because input data to encoder is reduce from 2 channel to 1 channel . 

why it is like this ?

**Comment by pschatzmann** [comment, marked as answer]
Maybe you should read the specification. 
What do you think this depends on ?
It is quite obvous that if you provide a different input you get a different result...

**Comment by aqildad-create** [comment]
I was expecting If I get 76 bytes compressed with 2  channel then with 1 channel 
It will be half of 76 but i get  72  .

So I personally concluded that SBC or other encoder with 2 channel is very
ideally  compressed just approximately 4 bytes higher  not like pcm that
its 2 channel is twice of single channel .

The four bytes difference is may be it contain information of channel .
